pages/cookbook/jettons.mdx

```diff
@@ -1,9 +1,161 @@
 # Fungible Tokens (Jettons)
 
-import { Callout } from 'nextra/components'
+import { Callout } from 'nextra-theme-docs';
 
-<Callout emoji="🚨">
+This page lists common examples of working with [jettons](https://docs.ton.org/develop/dapps/asset-processing/jettons).
 
-  This page is a stub. [Contributions are welcome!](https://github.com/tact-lang/tact-docs/issues)
+## Accepting jetton transfer
+
+Transfer notification message have the following structure.
+
+```tact
+message(0x7362d09c) TransferNotification {
+    queryId: Int as uint64;
+    jettonAmount: Int as coins;
+    fromAddress: Address;
+    forwardPayload: Slice as remaining;
+}
```

Comment:
I think it'll be better to name these fields as they're named in the [standard](https://github.com/ton-blockchain/TEPs/blob/master/text/0074-jettons-standard.md) itself, e.g. `amount` and `sender` instead of `jettonAmount` and `fromAddress`. This applies to other defined messages too.

Reply:
Resolved. Named message also as in TLB, with Jetton prefix